examples/06_assets/main.rs

```diff
@@ -117,6 +94,10 @@ impl State for Example {
 }
 
 fn main() {
+    let assets_path = format!("{}/examples/06_assets/resources/assets",
+                   env!("CARGO_MANIFEST_DIR"));
+    set_var("AMETHYST_ASSET_DIRS", assets_path);
```

Comment:
Also added some handling for setting up default asset directories. Do we want to demo that here, or revert this to setting up a `DirectoryStore` itself? It does seem kind of odd to have this example setting up asset loading differently from the others.